hs/src/Reach/Eval/Core.hs

```diff
@@ -3410,7 +3410,7 @@ evalPrim p sargs =
       vv <- case (vt, args) of
         (ST_Null, []) -> return $ SLV_Null at "variant"
         _ -> one_arg
-      void $ typeCheck_s CT_Assert vt vv
+      void $ typeCheck_s CT_Assert Nothing vt vv
```

Comment:
This should be the srcloc of the definition of the variant. I don't think that is actually tracked, but could be